sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobAudience.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * The audience to be used when requesting a token from Azure Active Directory (AAD).
+ * Note: This audience only has an effect when authenticating a TokenCredential.
+ */
+public class BlobAudience extends ExpandableStringEnum<BlobAudience> {
+
+    /**
+     * Gets default Audience used to acquire a token for authorizing requests to any Azure Storage account.
+     * If no audience is specified, this resource ID is the default value: "https://storage.azure.com/".
+     */
+    public static final BlobAudience PUBLIC_AUDIENCE = fromString("https://storage.azure.com/");
+
+    /**
+     * Creates a new instance of {@link BlobAudience} without a {@link #toString()} value.
+     * This constructor shouldn't be called as it will produce a {@link BlobAudience} which doesn't have a String enum
+     * value.
+     *
+     * @deprecated Use one of the constants or the {@link #fromString(String)} factory method.
+     */
+    @Deprecated
+    public BlobAudience() {
+    }
+
+    /**
+     * The service endpoint for a given storage account. Use this method to acquire a token for authorizing requests to
+     * that specific Azure Storage account and service only.
+     *
+     * @param storageAccountName The storage account name used to populate the service endpoint.
+     * @return the audience with the blob service endpoint.
+     */
+    public static BlobAudience getBlobServiceAccountAudience(String storageAccountName) {
```

Comment:
Change this to `create` to align with the name used in .NET.

Reply:
This change also needs to be applied to Data Lake, Queue, and File